certification/internal/shell/fake_skopeo_engine_test.go

```diff
@@ -0,0 +1,35 @@
+package shell
```

Comment:
Put this code into shell_suite_test.go, as was done for the Podman engine fakes.

Reply:
Just checked the fake podman and not sure if I understand what I shall put in shell_suite_test.go

Reply:
@bcrochet Just pulled the latest code and saw what you have done :) Will update the code